design-recipe-route.md

```diff
@@ -0,0 +1,107 @@
+# {{ METHOD }} {{ PATH}} Route Design Recipe
```

Comment:
I like that you planned out the routes too, it would have probably made my life a lot easier!